.github/workflows/build.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [main, ls-integration]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout liberty-tools-intellij
+        uses: actions/checkout@v3
+        with:
+          path: liberty-tools-intellij
+      - name: Checkout lsp4jakarta
+        uses: actions/checkout@v3
+        with:
+          repository: eclipse/lsp4jakarta
+          path: lsp4jakarta
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17
+          cache: 'maven'
+      - name: Cache Maven
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Cache Gradle
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Build Liberty-Tools-Intellij
+        working-directory: ./liberty-tools-intellij
+        run: |
+          gradle buildJakarta
+          gradle buildPlugin
```

Comment:
Should these use the project's Gradle wrapper?

Reply:
fixed!